iml-gui/crate/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+version = "0.1.0"
+name = "iml-gui"
+repository = "https://github.com/whamcloud/integrated-manager-for-lustre"
+authors = ["IML Team <iml@whamcloud.com>"]
+description = "The IML GUI"
+categories = ["wasm", "rust", "tailwindcss", "seed"]
+license = "MIT"
+readme = "./README.md"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dev-dependencies]
+wasm-bindgen-test = "0.2" # sync with `wasm-bindgen`
+
+[dependencies]
+chrono = { version = "0.4", features = ["wasmbind"] }
+js-sys = "0.3"
+seed = { git = "https://github.com/David-OConnor/seed", branch="master" }
+wasm-bindgen = "0.2" # sync with `wasm-bindgen-test`
+iml-wire-types = { path = "../../iml-wire-types", version = "0.2" }
+serde = { version = "1", features = ['derive'] }
+serde_json = "1.0"
+
+[dependencies.web-sys]
+version = "^0.3.28"
+features = [
+  "Navigator",
+  "EventSource",
+  "DomRect",
+  "Element",
+  "MessageEvent"
+]
+
+[profile.release]
+lto = true
```

Comment:
Does it make sense for wasm?

Reply:
Yes, doing this tells llvm to optimize .wasm file further. Ref: https://rustwasm.github.io/book/reference/code-size.html#compiling-with-link-time-optimizations-lto